tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -219,5 +236,25 @@ def handler(event: Dict[str, Union[str, int]],
     Raises:
         ExtractFilePathsError: An error occurred parsing the input.
     """
-    result = run_cumulus_task(task, event, context)
+    LOGGER.error(event)
+    try:
+        _VALIDATE_INPUT(event["event"]["payload"])
+    except JsonSchemaException as json_schema_exception:
+        LOGGER.error(json_schema_exception)
+        raise
+
+    try:
+        _VALIDATE_CONFIG(event["task_config"])
+    except JsonSchemaException as json_schema_exception:
+        LOGGER.error(json_schema_exception)
+        raise
+
+    result = task(event, context)
```

Comment:
Do we need o pass `context` to the `task` method? I am not seeing where we utilize this input anywhere in `task`. Recommend we remove it from the function call for task.

Reply:
will remove context from argument

Reply:
looks like `@LOGGER.inject_lambda_context` requires context argument in handler.
`decorate() missing 1 required positional argument: 'context'`

Keeping context as argument for now

Reply:
I am confused here. I agree you will need to keep context in handler since AWS needs that. I am referring to removing it from the `task` function definition. So `task(event, context) becomes `task(event)` and `def task(event, context): # pylint: disable-msg=unused-argument` becomes `def task(event):`

Reply:
oh I see. Removed context from task()